hail_scripts/v02/update_reference_data_in_existing_index.py

```diff
@@ -0,0 +1,142 @@
+import os
+import re
+
+os.system("pip install elasticsearch")
+
+import argparse
+from pprint import pprint
+import logging
+
+from hail_scripts.shared.elasticsearch_utils import ELASTICSEARCH_UPDATE
+
+import hail as hl
+from hail_scripts.v02.update_models.update_mt_schema import HGMDSchema, CLINVARSchema, CIDRSchema
+from hail_scripts.v02.utils.elasticsearch_client import ElasticsearchClient
+from lib.model.seqr_mt_schema import SeqrVariantsAndGenotypesSchema
+
+logger = logging.getLogger(__name__)
+
+# Change the paths to your hadoop/gcloud space
+clinvar_ht_path='/seqr-reference-data/GRCh37/clinvar/clinvar.GRCh37.ht'
```

Comment:
Hard-coding the paths is pretty convenient but non-standard. 

@knguyen142 Should we move all seqr-reference-data paths to the luigi config?

https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/master/luigi_pipeline/lib/global_config.py

Reply:
Those can be overridden https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/master/luigi_pipeline/configs/seqr-loading-local-GRCh37.cfg#L2-L3

Reply:
Ahh maybe I misunderstood your suggestion, what are you suggesting?

Reply:
Never mind, I just noticed [reference_ht_path](https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/master/luigi_pipeline/configs/seqr-loading-local-GRCh37.cfg#L11) is also in there and contains all the other ref datasets. Then this is the only script that needs to be updated.